src/machines/commandBarMachine.ts

```diff
@@ -288,7 +288,10 @@ export const commandBarMachine = setup({
           argConfig.skip ||
           (typeof argConfig.hidden === 'function'
             ? argConfig.hidden(context)
-            : argConfig.hidden)
+            : argConfig.hidden) ||
+          (typeof argConfig.required === 'function'
+            ? !argConfig.required(context)
+            : !argConfig.required)
```

Comment:
Looks like this was forgotten in #7506, `All arguments are skippable` should be true if hidden or skipeable or not required